cypress/support/parent-commands.ts

```diff
@@ -129,11 +131,32 @@ Cypress.Commands.add('checkInPageNavigationLinks', () => {
 // @TODO Check ComponentHeader <--- waiting for changes to this component to be merged.
 // Cypress.Commands.add('checkComponentHeader', () => {});
 
+Cypress.Commands.add('abortPrefetchRequests', (target) => {
```

Comment:
Do you mind explaining what this is doing?

Reply:
Oh yes, I totally forgot to add the docs to this command. Will add.